### PR TITLE
Add a "configuration" section to the JMX documentation

### PR DESCRIPTION
**Pull request by tonyzampogna**
This adds an example of how to put use the JMX registry. This may not be the best example, so I'm open to suggestions.

### CODE REVIEW

File: src/docs/implementations/jmx.adoc [hunk 1]
```diff
@@ -12,6 +12,34 @@ Micrometer also sometimes scrapes data from JMX beans for use in reporting metri
...
+counter = Counter.builder("app.counter.example.tick")
+        .tags("level", "normal")
+        .description("Number of info level events that made it to the logs")
+        .register(meterRegistry);
+
+counter.increment();
+
+----
+
+These values can be exposed to Prometheus using the https://github.com/prometheus/jmx_exporter[JMX Exporter].
```
Comment (jkschneider) [inline src/docs/implementations/jmx.adoc]: Why would I want to use the `jmx_exporter` as opposed to just exposing these metrics to Prometheus directly via `micrometer-registry-prometheus`. JMX will lose the fidelity of tags.

Comment (tonyzampogna) [reply src/docs/implementations/jmx.adoc]: Good question. I have a very customized use case. 

Remove it?

**Comment by jkschneider** [comment]
Do you think the counter example is covered in the [counters section](http://micrometer.io/docs/concepts#_counters)? The API usage for `Counter` is independent of the registry that it is being shipped to.

**Comment by tonyzampogna** [comment]
Good point, @jkschneider. If I take that out, it leaves just the JmxRegistry constructor. 

I’m new to Micrometer. I remember it was touch for me to figure this out, but I’m also ok with closing this PR. It really may not have much value.

**Comment by jkschneider** [comment]
Hmm. We probably need some "getting started" tutorials that take you from registering your first metric through visualizing it in the backend.

**Pull request closed without merging**